### DISCUSSION TITLE
chatglm3-6b 的function_call 是不是不支持对函数调用的结果做 进一步处理呢?

### DISCUSSION THREAD

**Discussion by adeagle** [zai-org/ChatGLM3#792] [General]
chatglm3-6b 的function_call 是不是不支持对函数调用的结果做 进一步的处理呢。比如我问 “查询北京的天气,仅输出温度信息
” 它的回答是 “根据您的要求，我已经成功查询了北京的天气信息。目前北京的温度为3℃，体感温度为-0℃。此外，当前湿度为18%，天气状况为晴朗。以上信息来自于可靠的天气API，数据准确可信”    但我只想它输出3℃

**Comment by zRzRzRzRzRzRzR** [comment]
不能，这个要自己写工程处理，glm3-6b却少对observation后的下一步反应
